enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/errorhandling/DefaultExceptionHandler.java

```diff
@@ -63,8 +63,8 @@ public class DefaultExceptionHandler {
     @ExceptionHandler(InvalidRequestObjectException.class)
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     public @ResponseBody ErrorResponse handleInvalidRequestException(InvalidRequestObjectException ex) {
-        logger.warn("Error occurred when registering to push server.", ex);
-        return new ErrorResponse("INVALID_REQUEST", "Invalid request object sent to Mobile Token API component.");
+        logger.warn("Error occurred when processing request object.", ex);
```

Comment:
Texting is confusing. The exception is related not only to push server or mtoken. It is thrown in `ActivationCodeService`, `PushRegistrationService` and `OnboardingServiceImpl`.